core/shared/protobuf.go

```diff
@@ -61,6 +61,10 @@ func (p *protoBufHandler) MountProtoBufConfigMapOnDataIndex(runtimeInstance api.
 
 	// Fetch Protobuf configmap for provided runtime
 	protoBufConfigMap, err := p.protoBufConfigMapHandler.FetchProtoBufConfigMap(runtimeInstance)
+	//changes
+	if protoBufConfigMaps == nil {
```

Comment:
@Prakritishrivastava probably there is a typo in the variable name, should you use protoBufConfigMap instead of protoBufConfigMaps?

Reply:
Thank you for pointing that out ,I will look into it....sorry about that.